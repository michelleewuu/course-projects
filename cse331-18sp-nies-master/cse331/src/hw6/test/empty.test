# Test the parser on empty file

LoadGraph empty empty.tsv

ListNodes empty

