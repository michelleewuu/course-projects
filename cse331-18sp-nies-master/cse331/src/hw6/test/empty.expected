# Test the parser on empty file

loaded graph empty

empty contains:

